pubtrawl.php

```diff
@@ -0,0 +1,66 @@
+<?php
```

Comment:
Thanks for giving it a better name. I didn't think `batch_test` was too bad. But this is miles better :+1: